Firestore/Source/API/FIRAggregateField.mm

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRAggregateField.h"
+
+#import "Firestore/Source/API/FIRAggregateField+Internal.h"
+#import "Firestore/Source/API/FIRFieldPath+Internal.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+#pragma mark - FIRAggregateField
+
+@interface FIRAggregateField ()
+@property(nonatomic, strong) FIRFieldPath *fieldPath;
+@property(nonatomic, readwrite) model::AggregateField::OpKind _op;
+- (instancetype)initWithFieldPath:(nullable FIRFieldPath *)fieldPath
+                           opKind:(model::AggregateField::OpKind)op;
+@end
+
+@implementation FIRAggregateField
+- (instancetype)initWithFieldPath:(nullable FIRFieldPath *)fieldPath
+                           opKind:(model::AggregateField::OpKind)op {
+  if (self = [super init]) {
+    self.fieldPath = fieldPath;
+    self._op = op;
+  }
+  return self;
+}
+
+@synthesize fieldPath;
+
+- (const std::string)name {
+  return model::AggregateField::OperatorKind([self _op]);
```

Comment:
I'm on the fence about this:
Remove the static function `static std::string OperatorKind(OpKind kind)` from `firebase::firestore::model::AggregateField`. And add a function in this `.mm` file that maps `_op` to a string. WDYT?

Reply:
I don't feel strongly about this either. I liked having the stringification with the enum, but since nothing else is using this code, I moved it into the `name` method.